test/Microsoft.AspNetCore.SignalR.Client.Tests/HttpConnectionTests.cs

```diff
@@ -118,9 +119,9 @@ public void CannotCreateConnectionWithNullUrlOnOptions()
                 // ignore connection error
             }
 
-            Assert.Equal(2, testSink.Writes.Count);
-            Assert.Equal("SendingHttpRequest", testSink.Writes[0].EventId.Name);
-            Assert.Equal("UnsuccessfulHttpResponse", testSink.Writes[1].EventId.Name);
+            Assert.Equal(2, testSink.Writes.Count());
+            Assert.Equal("SendingHttpRequest", testSink.Writes.First().EventId.Name);
```

Comment:
Is order important here?

Reply:
Can you convert to a list first up top then use that list?

Reply:
How about `var writes = testSink.Writes.ToList();` and then use the list rather than using linq everywhere?